src/layout/hooks/useHandleError.ts

```diff
@@ -29,7 +29,7 @@ export function useHandleError() {
   const router = useRouter();
   const pathname = usePathname()!;
 
-  const handleError = useEventCallback(
+  const handleError = useCallback(
```

Comment:
I don't know the reason for using `useEventCallback` here, but it was triggering errors in console quite often and prevented the issue at hand (error on vector store file creation) to be processed properly.